cmd/gke-gcloud-auth-plugin/cred.go

```diff
@@ -99,14 +101,16 @@ func main() {
 
 	var tokenProvider tokenProvider = nil
 	if *useEdgeCloud {
-		if *location == "" || *cluster == "" {
-			klog.Exit(fmt.Errorf("for --use_edge_cloud: --location and --cluster are required"))
+		if *project == "" || *location == "" || *cluster == "" {
```

Comment:
Won't making this field required break existing users?

Reply:
`gcloud edge-cloud container clusters get-credentials` does not yet utilize this tool.
This tool is not made to support any clients except gcloud, so breakage is not yet a concern.